client/src/pages/Dashboard.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+
+import { Grid } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+
+import Mentions from "./Mentions";
+import Platforms from "./Platforms";
+import Navbar from "./Navbar";
+
+const dashboardStyle = theme => ({
+  rightGridContainer: {
+    borderLeft: '1px solid black',
+    height: '100%'
+  }
+});
+
+class Dashboard extends Component {
+  state = {
+    platforms: [
+      {name: "Reddit", inUse: true},
+      {name: "Twitter", inUse: true},
+      {name: "Facebook", inUse: true},
+      {name: "Amazon", inUse: true},
+      {name: "Forbes", inUse: true},
+      {name: "Shopify", inUse: true},
+      {name: "Business Insider", inUse: true}
+    ],
+    mentions: [
+      {title: 'thing happened', platform: 'Reddit', desc: 'qwerty'},
+      {title: 'consume', platform: 'Forbes', desc: '12345'},
+      {title: 'buy', platform: 'Shopify', desc: 'abc123'},
+      {title: 'do', platform: 'Business Insider', desc: 'aedsfadwsf'}
+    ],
+    sort: 0
```

Comment:
Is this just a placeholder for now? I was thinking this could be implemented differently. Maybe a string value (e.g. date, title, author, etc.)? Or an object with key value pairs with keys as sort type, and values as booleans. Or even an array with 1s and 0s combined with a reducer to handle which sort method to use.

Reply:
Yeah, it's just a placeholder for now to just display those text